seqstats/seqstats.go

```diff
@@ -100,6 +107,7 @@ func main() {
 		csum = seqlens[i] + csum
 	}
 	b.avg = float64(b.size) / float64(b.totSeqs)
-	// Print the statistics of the assembly as key-value pairs.
+	b.perGC = float64(ctr["G"]+ctr["C"]) / float64(b.size) * 100
```

Comment:
This is not formally correct for genomes with ambiguous letters. The denominator should be `float64(ctr['a']+ctr['c']+ctr['g']+ctr['t'])`.